test/spec/platform_spec.rb

```diff
@@ -85,14 +85,15 @@
 		end
 	end
 	
-	describe "Composer Plugin" do
+	describe "Composer Plugin", :focused => true do
```

Comment:
stray `:focused`

Reply:
Damn it, I even had that change in a separate commit so exactly this would not happen, but accidentally squashed it rather than discarding on final rebase ;)